File: go/1099.go
package main

import "fmt"

func main() {
  var n, x, y, maior, menor, sum int
  fmt.Scanf("%d", &n)
  for i := 0; i < n; i++ {
    fmt.Scanf("%d", &x)
    fmt.Scanf("%d", &y)
    maior = x
    menor = y
    if y > x {
      maior = y
      menor = x
    }
  
    sum = 0
    for j := menor + 1; j < maior; j++ {
      if j % 2 == 0 {
        continue
      }
      sum += j
    }

    fmt.Println(sum)
  }
}

File: lua/1099.lua
local n, x, y, maior, menor, sum = 0, 0, 0, 0, 0, 0

n = io.read("*n")

for _ = 1, n do
    x = io.read("*n")
    y = io.read("*n")

    if y > x then
        maior = y
        menor = x
    else
        maior = x
        menor = y
    end

    sum = 0
    for j = menor + 1, maior - 1 do
        if j % 2 == 1 then
            sum = sum + j
        end
    end

    print(sum)
end
